README.md

```diff
@@ -100,31 +98,18 @@ qiime chemistree predict-fingerprints --p-sirius-path 'sirius-osx64-4.0.1/bin' \
   ```
 
 This gives us a QIIME 2 artifact of type `CSIFolder` that contains probabilities of molecular substructures (total 2936 molecular properties) within in each feature.
-We now generate a contingency table with these probabilities i.e. molecular fingerprints of MS1 features in our experiment. This is of type `FeatureTable[Frequency]`.
-
-```bash
-qiime chemistree collate-fingerprint --i-csi-result fingerprints.qza \
---p-qc-properties \
---o-collated-fingerprints collated_fingerprints_qc.qza
-```
-
-By default, we only use PUBCHEM fingerprints (total 489 molecular properties). Adding `--p-no-qc-properties` retains all (2936) the molecular properties in the contingency table. This table is used to generate out hierarchy of molecules!
+Now, we use these predicted molecular substructures to generate a hierarchy of molecules as follows:
 
 ```bash
 qiime chemistree make-hierarchy \
-  --i-collated-fingerprints collated_fingerprints_qc.qza \
-  --p-prob-threshold 0.5 \
-  --o-tree demo-chemisTree.qza
-```
-
-This generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`. **Note**: SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in your experiment (based on factors such as sample type, the quality MS2 spectra, and used-defined tolerances such as `--p-ppm-max`, `--p-zodiac-threshold`). Thus, we need to remove the MS1 features without fingerprints from the feature table with:
-
-```bash
-qiime chemistree match-table --i-tree demo-chemisTree.qza \
+  --i-csi-result fingerprints.qza \
   --i-feature-table feature-table.qza \
-  --o-filtered-feature-table filtered-feature-table.qza
+  --o-tree demo-chemisTree.qza \
+  --o-matched-feature-table filtered-feature-table.qza
 ```
 
-This filters the MS1 table to include only the MS1 features with molecular fingerprints. The resulting table is also of type `FeatureTable[Frequency]`.
+This method performs two tasks:
+1. Generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`. By default, we only use PUBCHEM fingerprints (total 489 molecular properties). Adding `--p-no-qc-properties` retains all (2936) the molecular properties in the contingency table.

```

Comment:
Is there a version for PUBCHEM?

Reply:
It keeps getting updated - the latest stable release was downloaded on 13 August 2017. I'll add a note noting the same.